Source/WebCore/platform/graphics/gstreamer/GStreamerVideoFrameConverter.cpp

```diff
@@ -50,14 +50,7 @@ GStreamerVideoFrameConverter::GStreamerVideoFrameConverter()
     auto videoconvert = makeGStreamerElement("videoconvert", nullptr);
     auto videoscale = makeGStreamerElement("videoscale", nullptr);
     m_sink = makeGStreamerElement("appsink", nullptr);
-
-    if (webkitGstCheckVersion(1, 24, 0)) {
-        g_object_set(m_sink.get(), "emit-signals", TRUE, nullptr);
-        g_signal_connect(m_sink.get(), "propose-allocation", G_CALLBACK(+[](GstElement*, GstQuery* query) -> gboolean {
-            gst_query_add_allocation_meta(query, GST_VIDEO_META_API_TYPE, nullptr);
-            return TRUE;
-        }), nullptr);
-    }
```

Comment:
Why do you remove this?

Reply:
That's no longer needed since we manually add the video meta afterwards.